### DISCUSSION TITLE
I GET DIFFERENT OUTPUT FOR -config & -theme options FOR SAME  CONFIG file

### DISCUSSION THREAD

**Discussion by fr05t-byt3** [davatorium/rofi#1844] [Q&A]
![-theme](https://github.com/davatorium/rofi/assets/74895351/8368b713-5cc5-4c44-b59a-dff9fb807e13)
^^^^rofi -show drun -theme ./config2.rasi^^^^
![-config](https://github.com/davatorium/rofi/assets/74895351/9870317c-39b9-4f6d-aa53-b73d3da051c1)
^^^^rofi -show drun -config ./config2.rasi^^^^

configfile (https://gist.github.com/Frostbyte999/5a7d675ed3bcf1d36771afc28ab863bd)

ROFI VERSION : Version: 1.7.3
I Use WSL with Kali Linux

**Comment by DaveDavenport** [comment]
Yes, this is correct. 

Try adding `-dump-theme` to see the difference.
-theme unloads all theme data and loads your theme.
-config loads the theme on top of the existing (default) theme.

**Reply by fr05t-byt3** [reply]
should redirect the output of rofi -dump-theme ./Desktop/config2.rasi to config.rasi ???

**Reply by DaveDavenport** [reply]
No, that won't work, see my remark above.
What I would do:  
copy your config.rasi to  ~/.config/rofi/mytheme.rasi and remove the configuration part.

then remove the theme part from your config.rasi and append:
```css
@theme "mytheme"
```
This (see documentation) will *replace* the default theme, by mytheme.

It is going wrong because you load both the default theme as your theme, therefor you are seeing elements of both!
As explained in documentation.

**Reply by DaveDavenport** [reply]
https://davatorium.github.io/rofi/current/rofi-theme.5/#multiple-file-handling
